Source/WebCore/css/CSSCounterValue.cpp

```diff
@@ -53,8 +53,9 @@ bool CSSCounterValue::equals(const CSSCounterValue& other) const
 
 String CSSCounterValue::customCSSText() const
 {
-    auto listStyleSeparator = m_counterStyle->valueID() == CSSValueDecimal ? ""_s : ", "_s;
-    auto listStyleLiteral = m_counterStyle->valueID() == CSSValueDecimal ? ""_s : counterStyleCSSText();
+    bool isDecimal = m_counterStyle->valueID() == CSSValueDecimal || (m_counterStyle->isCustomIdent() && m_counterStyle->customIdent() == "decimal"_s);
```

Comment:
Should this be ASCII case insensitive? Typically I like to add a test case for that just to confirm. Why isn't the custom value parsed in some way? Seems strange to have this explicit check needed only in serialization without shared code since you'd think we'd need to code that uses this to render would need the same logic.

Reply:
I added a testcase, and we don't need a case-insensitive comparison here. I presume we have some code somewhere that lowercases custom-idents.